### DISCUSSION TITLE
Issue with Published sensor data to AllThingsTalk

### DISCUSSION THREAD

**Discussion by serbiansky** [DesconBelgrade/Klimerko#36] [Q&A] [answered]
All of the sudden on my Klimerko I notice that Published sensor data to AllThingsTalk is missing, Published diagnostic data to AllThingsTalk is working fine but not sensor one. I notice only one upload in 10-15 min and the error is [MQTT] Lost Connection. Reason: -4.
I tried to reconnect to wifi, factory reset and no luck. Attached is a log from the serial monitor. Any ideas?
[Log.txt](https://github.com/DesconBelgrade/Klimerko/files/9524390/Log.txt)

**Comment by epiller** [comment, marked as answer]
There was an issue with some MQTT topics on AllThingsTalk Platform - it should be resolved now!
